src/_versions.ts

```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.2.2',
```

Comment:
this version should change? (from package.json)

Reply:
Done